### PR TITLE
Release v202404.0.0

### PR DESCRIPTION
**Pull request by MakisH**
Currently affected by https://github.com/precice/vm/issues/92

### CODE REVIEW

File: provisioning/.alias [hunk 1]
```diff
@@ -1,12 +1,12 @@
 # preCICE config visualizer
 function preciceToPNG(){
-  cat "${1:-precice-config.xml}" | precice-config-visualizer | dot -Tpng > precice-config.png
+  precice-config-visualizer "${1:-precice-config.xml}" | dot -Tpng > precice-config.png
```
Comment (fsimonis) [inline provisioning/.alias:3]:
```suggestion
  precice-config-visualizer -o precice-config.png "${1:-precice-config.xml}"
```

File: provisioning/.alias [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 }
 
 function preciceToPDF(){
-  cat "${1:-precice-config.xml}" | precice-config-visualizer | dot -Tpdf > precice-config.pdf
+  precice-config-visualizer "${1:-precice-config.xml}" | dot -Tpdf > precice-config.pdf
```
Comment (fsimonis) [inline provisioning/.alias:7]:
```suggestion
  precice-config-visualizer -o precice-config.pdf "${1:-precice-config.xml}"
```

File: provisioning/.alias [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 }
 
 function preciceToSVG(){
-  cat "${1:-precice-config.xml}" | precice-config-visualizer | dot -Tsvg > precice-config.svg
+  precice-config-visualizer "${1:-precice-config.xml}" | dot -Tsvg > precice-config.svg
```
Comment (fsimonis) [inline provisioning/.alias:11]:
```suggestion
  precice-config-visualizer -o precice-config.svg "${1:-precice-config.xml}"
```

File: provisioning/install-precice.sh [hunk 4]
```diff
@@ -14,10 +14,10 @@ fi
     cd precice
     git pull
     mkdir -p build && cd build/
-    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=Debug -DBUILD_TESTING=OFF -Wno-dev ..
+    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTING=OFF -Wno-dev ..
```
Comment (fsimonis) [inline provisioning/install-precice.sh:17]: To turn the debug log on.
Is there a reason to include debug info?

```suggestion
    cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DPRECICE_RELEASE_WITH_DEBUG_LOG=ON -DBUILD_TESTING=OFF -Wno-dev ..
```

File: provisioning/install-precice.sh [hunk 5]
```diff
@@ -14,10 +14,10 @@ fi
...
     make -j "$(nproc)"
     rm -fv ./*.deb && make package
-    sudo apt-get install -y ./libprecice2_*.deb
+    sudo apt-get install -y ./libprecice3_*.deb
```
Comment (fsimonis) [inline provisioning/install-precice.sh:20]: Any reason not to keep this version agnostic?

```suggestion
    sudo apt-get install -y ./libprecice*_*.deb
```

File: provisioning/install-precice.sh [hunk 6]
```diff
@@ -60,20 +62,7 @@ pip3 install --upgrade pip
...
-(
-    cd tutorials/perpendicular-flap/fluid-nutils/
-    python3 -m venv nutils6-env
-    # shellcheck source=/dev/null
-    source nutils6-env/bin/activate
-    pip3 install nutils==6.3 pyprecice
-    sed -i "s/python3/nutils6-env\/bin\/python3/g" ./run.sh
-    deactivate
-)
+pip3 install --user pandas matplotlib # Needed for the post-processing scripts
```
Comment (fsimonis) [inline provisioning/install-precice.sh:65]: polars is needed by precice-profiling

```suggestion
pip3 install --user pandas matplotlib polars # Needed for the post-processing scripts
```

File: provisioning/install-precice.sh (hunk 4 above)
Comment (MakisH) [reply provisioning/install-precice.sh:17]: This originates from this (maybe outdated) suggestion: https://github.com/precice/vm/issues/41

Comment (MakisH) [reply provisioning/install-precice.sh:17]: I think the intention was always that we might want to use the VM for debugging some cases involving difficult-to-install solvers.

**Review by fsimonis — APPROVED** [review]
Last comment from my side

**Pull request merged**